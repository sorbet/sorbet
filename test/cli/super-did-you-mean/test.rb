# typed: strict
class Module; include T::Sig; end

class Parent
  sig {void}
  def supper
    # Previously, Sorbet would include this in the "did you mean" suggestions,
    # because `supper` is like `<super>`
  end
end

class Child < Parent
  sig {void}
  def no_parent_method
    super
  end
end
