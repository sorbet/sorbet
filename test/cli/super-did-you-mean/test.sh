#!/bin/bash

main/sorbet --silence-dev-message test/cli/super-did-you-mean/test.rb 2>&1
