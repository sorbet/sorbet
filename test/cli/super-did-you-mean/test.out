test/cli/super-did-you-mean/test.rb:15: Method `no_parent_method` does not exist on ancestors of `Child` https://srb.help/7048
    15 |    super
            ^^^^^
  Note:
    For help fixing `super` errors: https://sorbet.org/docs/typed-super
Errors: 1
